src/client.js

```diff
@@ -65,6 +66,15 @@ function fetchResourcesForProducts(products, client) {
   }, []);
 }
 
+function adaptConfig(config) {
```

Comment:
Since you're only normalizing one field, why not just have a `normalizeAccessToken` method? that receives the config, and returns whichever token, as well as logging a deprecation notice for the deprecated config value.

That way, the transformation isn't as opaque in the build client method.

Also, wasn't there some resolution about this being only `accessToken`? @swalkinshaw can you fill me in with what happened here?